man/systemd-system.conf.xml

```diff
@@ -215,6 +215,17 @@
         good.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>NoNewPrivileges=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, ensures
+        that PID 1 and all its children can never gain new privileges
+        through <function>execve()</function> (e.g. via setuid or
+        setgid bits, or filesystem capabilities). Defaults to false.
+        Also see <ulink url="https://www.kernel.org/doc/html/latest/userspace-api/no_new_privs.html">No New Privileges Flag</ulink>.
```

Comment:
could you add a brief explanation that making use of this is problematic on general purpose distributions, i.e. that most systems are not ready for running with this turned on? This can be done in some vague, generic words, but it should be mentioned, so that people don't mindlessly turn this on under the assumption this will just work.